Doc/library/dis.rst

```diff
@@ -960,22 +998,19 @@ All of the following opcodes use their arguments.
    Loads the global named ``co_names[namei]`` onto the stack.
 
 
-.. opcode:: SETUP_LOOP (delta)
-
-   Pushes a block for a loop onto the block stack.  The block spans from the
-   current instruction with a size of *delta* bytes.
-
+.. opcode:: SETUP_FINALLY (delta)
 
-.. opcode:: SETUP_EXCEPT (delta)
+   Pushes a try block from a try-finally or try-except clause onto the block
+   stack.  *delta* points to the finally block or the first except block.
 
-   Pushes a try block from a try-except clause onto the block stack. *delta*
-   points to the first except block.
 
+.. opcode:: CALL_FINALLY (delta)
 
-.. opcode:: SETUP_FINALLY (delta)
+   Pushes the address of the next instruction onto the stack and increments
+   bytecode counter by *delta*.  Used for calling the finally block as a
+   "subroutine".
 
-   Pushes a try block from a try-except clause onto the block stack. *delta*
-   points to the finally block.
+   .. versionadded:: 3.7
```

Comment:
We don't seem to be adding `versionadded` for other new opcodes, do we?

Reply:
See `SETUP_ASYNC_WITH`. Or the 2.7 docs, which almost always omit it.

Reply:
I think it should be added for `SETUP_ASYNC_WITH`. It is used with other new opcodes added before and after `SETUP_ASYNC_WITH`.